examples/GPU/example_3d_trajectory_display.py

```diff
@@ -0,0 +1,117 @@
+"""
+======================
+3D Trajectory display
+======================
```

Comment:
Why is it named 3D trajectory display when we only show 2D k-space slices ?

Reply:
Its a 3D trajectory..
Renaming for clarity

Reply:
Please don't mark general comments as resolved as an assignee, it's difficult for reviewers if they have to look into each solved comment just to be sure it was really solved or simply to read the assignee's answer

Reply:
I agree, i didnt resolve any usually (Unlike @paquiteau :P ), I think github does something automatically I am not sure..